build-system/tasks/report-test-status.js

```diff
@@ -38,7 +39,7 @@ const TEST_TYPE_SUBTYPES = isGithubActionsBuild()
       ['integration', ['firefox', 'safari', 'edge', 'ie']],
       ['unit', ['firefox', 'safari', 'edge']],
     ])
-  : isTravisBuild()
+  : isCircleciBuild()
```

Comment:
Wasn't the plan to use a generic name like `isCiBuild`?

Reply:
`TEST_TYPE_SUBTYPES` determines which tests run on which service, so it needs the specific name. (E.g. see `isGithubActionsBuild()` on line 37.)

We do use the generic name everywhere else :)